app/views/vendors/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<table id="vendor_show_page" class="generic_datatable table table-striped table-bordered">
+  <caption><h1><%= @vendor.name %></h1></caption>
+
+  <thead>
+    <tr>
+      <th>List of <%= @vendor.name %> Items</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @items.each do |item| %>
+      <tr>
+        <td><%= item.name%></td>
```

Comment:
hmmmm it would be more useful if these were clickable links to the items, right?